scss/common/_base.scss

```diff
@@ -418,59 +418,63 @@
         opacity: .3;
     }
 
-    $loading-stroke-width: 2px !default;
-    $loading-size: $icon-size - $loading-stroke-width !default;
-    $loading-speed: 2s !default;
-    $loading-color: $accent;
-    $loading-outer-size: ($loading-size + ($loading-stroke-width * 2));
-    $loading-half-size: ($loading-outer-size / 2) !default;
-
-    .k-icon.k-i-loading {
+    // Loading indicator
+    $loading-size: 1em !default;
+    $loading-base-font-size: 16 !default;
+    $loading-base-font-size-px: 16px !default;
+    $loading-font-size: 16 !default;
+    $loading-font-size-px: 16px !default;
+    $loading-stroke-width: calc( .03 *( #{$loading-font-size} / #{$loading-base-font-size}) * 1px) !default;
```

Comment:
As far as I recall, @yordanov wanted these to change based on the loading indicator size. If this is calculated in `em`s, it will follow the `font-size` set to the `k-i-loading` element.